python/Ganga/GPIDev/Schema/Schema.py

```
@@ -243,14 +248,18 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
                 defvalue = _found_attrs[stored_attr_key]
             else:
                 defvalue = config[attr]
-                _found_attrs[stored_attr_key] = defvalue
+                from Ganga.GPIDev.Base.Proxy import stripProxy
+                ## Just in case a developer puts the proxied object into the default value!
+                _found_attrs[stored_attr_key] = stripProxy(defvalue)
```

Does this ever actually happen? It really shouldn't and if it does we should raise an exception and tell the developer off.